problems/200-299/problem-291.js

```diff
@@ -8,4 +8,41 @@
  * For example, given a population with weights [100, 200, 150, 80] and a boat limit of 200, the smallest
  * number of boats required will be three.
  */
-// TODO:
+const removeElement = (array, limit) => {
+    let j = array.length - 1;
+
+    while(j >= 0) {
+    if (array[j] > limit) {
+      array.splice(j, 1);
+      console.log("here", array)
+    }
+    --j;
+    }
+    return array;
+}
+
+const mainFunc = (array, limit) => {
+  let counter = 0;
+  array.sort(function(a, b){return a - b});
+  array = removeElement(array, limit);
```

Comment:
I think you can use the built-in function `filter` to do this. No need to write your own function here

Reply:
Yes, I just fixed that. One line of code did the job 👍🏻

Reply:
Have you pushed your change yet? I don't see the update

Reply:
I see you created a second PR to fix this. I think you only need to push to the same PR when you address the comment. Thanks

Reply:
Yes. I did a mistake and didn't wanna leave that PR empty. Thanks 👍👍